nni/nas/hub/pytorch/utils/pretrained.py

```diff
@@ -41,7 +41,16 @@
     # autoformer
     'autoformer-tiny': f'{NNI_BLOB}/nashub/autoformer-searched-tiny-1e90ebc1.pth',
     'autoformer-small': f'{NNI_BLOB}/nashub/autoformer-searched-small-4bc5d4e5.pth',
-    'autoformer-base': f'{NNI_BLOB}/nashub/autoformer-searched-base-c417590a.pth'
+    'autoformer-base': f'{NNI_BLOB}/nashub/autoformer-searched-base-c417590a.pth',
```

Comment:
what are the above three checkpoints used for?

Reply:
Oh, it's outdated model weights file, forgot to delete. I'll delete it later.